src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -1,35 +1,79 @@
+import { getIdentifierFromId } from '../../../utils/general-helpers';
+import { getFullCristinName } from '../../../utils/user-helpers';
 import { StyledStatusMessageBox } from '../../messages/components/PublishingRequestMessagesColumn';
 
 interface DeletedRegistrationInformationProps {
   registration: Registration;
+  unpublishingNote: PublicationNote;
 }
 
-export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+function extractId(createdBy: string) {
+  return '/cristin/person/' + createdBy.split('@')[0];
+}
+
+export const DeletedRegistrationInformation = ({
+  registration,
+  unpublishingNote,
+}: DeletedRegistrationInformationProps) => {
   const { t } = useTranslation();
+
+  const duplicateRegistrationIdentifier = getIdentifierFromId(registration.duplicateOf ?? '');
+
   const duplicateRegistrationQuery = useQuery({
-    enabled: !!registration.duplicateOf,
-    queryKey: ['registration', registration.duplicateOf],
-    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    enabled: !!duplicateRegistrationIdentifier,
+    queryKey: ['registration', duplicateRegistrationIdentifier],
+    queryFn: () => fetchRegistration(duplicateRegistrationIdentifier),
     meta: { errorMessage: t('feedback.error.get_registration') },
   });
 
+  const cristinIdentifier = unpublishingNote.createdBy ? extractId(unpublishingNote.createdBy) : null;
+  const personQuery = useQuery({
+    queryKey: ['cristinPerson', cristinIdentifier],
+    queryFn: () => fetchPerson(cristinIdentifier ?? ''),
+    enabled: !!cristinIdentifier,
+    meta: { errorMessage: t('feedback.error.get_registration') },
+  });
+
+  const person = personQuery.data;
+
   const duplicateRegistrationTitle = duplicateRegistrationQuery.data?.entityDescription?.mainTitle;
-  const unpublishingNote = registration.publicationNotes?.find((note) => note.type === 'UnpublishingNote');
 
   return (
     <StyledStatusMessageBox
       sx={{
         bgcolor: 'publishingRequest.main',
       }}>
-      {(registration.status === RegistrationStatus.Unpublished ||
-        registration.status === RegistrationStatus.Deleted) && (
+      {registration.status === RegistrationStatus.Deleted ? (
         <Typography>{t(`registration.status.${registration.status}`)}</Typography>
+      ) : (
+        <Typography>{t(`registration.status.UNPUBLISHED`)}</Typography>
       )}
+      <Box sx={{ display: 'flex', flexDirection: 'row', minWidth: '6rem', gap: '0.5rem', alignItems: 'center' }}>
+        {personQuery.isFetching ? (
+          <Skeleton variant="circular" />
+        ) : (
+          <>
+            {person && (
+              <ProfilePicture
+                sx={{ width: '1.5rem', height: '1.5rem' }}
+                fullName={getFullCristinName(person?.names)}
+                personId={person?.id ?? ''}
+              />
+            )}
+          </>
+        )}
```

Comment:
Egentlig litt småskeptisk til å ta inn person-visning her, siden det nå bare vises for en bestemt type publiseringsevent. Tror brukere vil forvente at alle sånne "event-bokser" skal oppøre seg likt og vise samme data, så når person vises for avpublisering, men ikke publisering av metadata/filer, tildelig av DOI, etc, så vil det fort oppfattes som bugs alle steder det ikke vises, i stedet for en feature der det vises. Men om de ønsker det sånn nå skal jeg ikke legge meg borti det 😅 

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/8cf22eb5-4ac6-48ac-afc4-47308ed7810e)

Reply:
I tegningene så er det også bilde av person på tickets: 
![image](https://github.com/BIBSYSDEV/NVA-Frontend/assets/44363838/6097328a-f5cb-4e59-ab4b-59e8d93a1ac1)

Reply:
Stokket om på rekkefølgen. Det gjør vondt å se på.

Reply:
Jepp, bare frykter at brukere kommer til å oppleve det som en bug at det ikke vises noen andre plasser enn for avpublisert, heller enn en feature akkurat der. Men om de ønsker at det skal vises bare for avpublisert nå skal jeg ikke legge meg bort det :p